app/views/research_sessions/preview.html.erb

```diff
@@ -66,7 +66,7 @@
       <%= @research_session.methodology_list %>
     <% end %>
     <p>
-      With your permission we will record the session using
+      With your permission we will record the <%= 'your child\'s/the child\'s in your care' if @research_session.unable_to_consent? %> session using
```

Comment:
Need to lose the "the" in the case of `unable_to_consent?` as this'll result in "the your child's".

You can also switch to double quotes here to avoid escaping the apostrophes. Or better still, go to a smart quote: "child‘s" (ALT-])